tools/src/tester/ResultLogger.ts

```diff
@@ -29,15 +29,17 @@ export class ConsoleResultLogger implements ResultLogger {
   }
 
   log (evaluation: StoryEvaluation): void {
+    const with_padding = [Result.FAILED, Result.ERROR].includes(evaluation.result) || this._verbose
+    if(with_padding) console.log()
```

Comment:
Lack of space after `if` looks suspicious, missing linter?

Reply:
Same issue with the previous PR: the rule was moved into a plugin.
I've resolved.